packages/snap-preact/components/src/components/Molecules/Select/Select.tsx

```diff
@@ -243,7 +226,10 @@ export const Select = observer((properties: SelectProps): JSX.Element => {
 	// options can be an Array or ObservableArray - but should have length
 	return typeof options == 'object' && options?.length ? (
 		<CacheProvider>
-			<div {...styling} className={classnames('ss__select', { 'ss__select--disabled': disabled }, className)}>
+			<div
+				{...styling}
+				className={classnames('ss__select', native ? 'ss__select--native' : 'ss__select--nonnative', { 'ss__select--disabled': disabled }, className)}
```

Comment:
Doesn't seem like we need both `native` and `nonnative` modifiers. Let's just use `native` perhaps?